deltacat/tests/compute/test_compact_partition_rebase_then_incremental.py

```diff
@@ -0,0 +1,347 @@
+    """
+    compacted_table_stream: Stream = ds.get_stream(
+        namespace=compacted_delta_locator.namespace,
+        table_name=compacted_delta_locator.table_name,
+        table_version=compacted_delta_locator.table_version,
+        **ds_mock_kwargs,
+    )
+    test_table: pa.Table = pa.Table.from_arrays(
+        incremental_deltas_arrow_arrays_param,
+        names=column_names_param,
+    )
+    staged_partition: Partition = ds.stage_partition(
+        compacted_table_stream, partition_values_param, **ds_mock_kwargs
+    )
+    ds.commit_delta(
+        ds.stage_delta(test_table, staged_partition, **ds_mock_kwargs), **ds_mock_kwargs
+    )
+    ds.commit_partition(staged_partition, **ds_mock_kwargs)
+    source_table_stream_after_committed: Stream = ds.get_stream(
+        namespace=compacted_delta_locator.namespace,
+        table_name=compacted_delta_locator.table_name,
+        table_version=compacted_delta_locator.table_version,
+        **ds_mock_kwargs,
+    )
+    source_partition: Partition = ds.get_partition(
+        source_table_stream_after_committed.locator,
+        partition_values_param,
+        **ds_mock_kwargs,
+    )
+    destination_table_stream: Stream = ds.get_stream(
+        destination_table_namespace,
+        destination_table_name,
+        destination_table_version,
+        **ds_mock_kwargs,
+    )
+    destination_partition_locator: PartitionLocator = PartitionLocator.of(
+        destination_table_stream.locator,
+        partition_values_param,
+        None,
+    )
+    compact_partition_params = CompactPartitionParams.of(
+        {
+            "compaction_artifact_s3_bucket": TEST_S3_RCF_BUCKET_NAME,
+            "compacted_file_content_type": ContentType.PARQUET,
+            "dd_max_parallelism_ratio": 1.0,
+            "deltacat_storage": ds,
+            "deltacat_storage_kwargs": ds_mock_kwargs,
+            "destination_partition_locator": destination_partition_locator,
+            "hash_bucket_count": hash_bucket_count_param,
+            "last_stream_position_to_compact": source_partition.stream_position,
+            "list_deltas_kwargs": {**ds_mock_kwargs, **{"equivalent_table_types": []}},
+            "pg_config": pgm,
+            "primary_keys": primary_keys_param,
+            "rebase_source_partition_locator": None,
+            "records_per_compacted_file": records_per_compacted_file_param,
+            "s3_client_kwargs": {},
+            "source_partition_locator": source_partition.locator,
+            "sort_keys": sort_keys if sort_keys else None,
+        }
+    )
+    rcf_file_s3_uri = compact_partition_func(compact_partition_params)
+    compacted_delta_locator = get_compacted_delta_locator_from_rcf(
+        setup_s3_resource, rcf_file_s3_uri
+    )
+    tables = ds.download_delta(compacted_delta_locator, **ds_mock_kwargs)
+    compacted_table = pa.concat_tables(tables)
+    assert compacted_table.equals(
+        expected_terminal_compact_partition_result
```

Comment:
same here regarding combine_chunks and sort.

Reply:
Done